index.html

```diff
@@ -0,0 +1,162 @@
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- header -->
+    <header class="w-full h-20 shadow bg-slate-900 fixed z-50">
+      <div class="flex justify-around align-items-center h-full w-full">
+        <!-- header-left (logo, nav-links) -->
+        <div class="flex justify-center align-items-center">
+          <!-- Logo -->
+          <a href="/index.html">
+            <h5
+              class="h5 cursor-pointer mr-10 text-cyan-300 logo text-small text-italic inline-flex justify-center align-items-center"
+            >
+              <!-- <img src="" alt="logo" class="w-14 h-14" /> -->
+              QUIZ<span class="text-white">mania</span>
+            </h5>
+          </a>
+          <!-- Logo -->
+
+          <!-- Links -->
+          <a href="/index.html">
+            <h6 class="mr-6 cursor-pointer h6 text-cyan-300">Home</h6>
+          </a>
+          <a href="/pages/product-listing-page/product-listing-page.html">
+            <h6 class="mr-6 cursor-pointer h6 text-hover-rose-500 text-white">
+              Quizzes
+            </h6>
+          </a>
+          <!-- Links -->
+        </div>
+        <!-- header-left (logo, nav-links) -->
+
+        <!-- header middle (searchbar) -->
+        <form action="" class="search__bar__wrapper">
+          <input type="text" class="search__bar" placeholder="Search" />
+          <a href="#">
+            <i class="fas fa-search search__icon text-cyan-500"></i>
+          </a>
+        </form>
+        <!-- header middle (searchbar) -->
+
+        <!-- header right (nav-links, login/signup CTA) -->
+        <div class="flex justify-center align-items-center">
+          <!-- navigation links -->
+          <a href="/pages/wishlist-page/wishlist-page.html">
+            <span class="w-full h-full tooltip btn-icon mr-5">
+              <i
+                class="text-cyan-500 text-hover-cyan-300 fas fa-user text-3xl"
+              ></i>
+
+              <span
+                class="tooltip-text text-black text-lg tooltip-text-bottom bg-cyan-100"
+              >
+                Profile
+              </span>
+            </span>
+          </a>
+
+          <!-- login/signup CTA -->
+          <a
+            class="btn btn-square-solid btn-md text-white bg-cyan-600 btn-bg-hover-cyan-300 mr-7"
+          >
+            LOGIN
+          </a>
+
+          <!-- navigation links -->
+        </div>
+        <!-- header right (nav-links, login/signup CTA) -->
+      </div>
+    </header>
+    <!-- header -->
+
+    <!-- main section -->
+    <main
+      class="relative w-full flex align-items-center justify-center"
+      style="top: 5rem; min-height: 100vh"
+    >
+      <div style="max-width: 980px; margin: auto">
+        <div class="flex flex-col justify-center mb-10">
+          <h2 class="mb-10 text-8xl text-cyan-600">Quiz Mania</h2>
+
+          <h5 class="text-xl">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Assumenda,
+            <br />
```

Comment:
why are we using break here ? You can use ch unit with container width to restrict according to number of characters.